app/components/publications/publication/translations/translations-info-panel.hbs

```diff
@@ -0,0 +1,96 @@
+<Auk::Panel class={{if this.isEditing "auk-panel--editing"}}>
+  <Auk::Panel::Header>
+    <Auk::Toolbar>
+      <Auk::Toolbar::Group @position="left">
+        <Auk::Toolbar::Item>
+          <h4 class="auk-panel__title">
+            {{t "translation-info"}}
+          </h4>
+        </Auk::Toolbar::Item>
+      </Auk::Toolbar::Group>
+      <Auk::Toolbar::Group @position="right">
+        <Auk::Toolbar::Item>
+          <Auk::Button
+            @skin="tertiary"
+            @icon="pencil"
+            @disabled={{not (or this.publicationStatus.isPending this.publicationStatus.isPaused)}}
```

Comment:
Dit lijkt mij een vreemde conditie. Nu kan de edit mode enkel geopend worden wanneer de status 'opgestart', 'teruggetrokken' of 'gepubliceerd' is?  Ik zou de voorwaarde eerlijk gezegd volledig weglaten op dit moment.

Misschien dat ze wel willen disablen bij `publicationStatus.isFinal` (= teruggetrokken of gepubliceerd). Maar dan zou ik een nieuwe story maken om dat gedrag consistent in alle panels toe te voegen.